tsfeatures/tsfeatures.py

```diff
@@ -964,76 +971,14 @@ def tsfeatures(ts: pd.DataFrame,
         Pandas DataFrame where each column is a feature and each row
         a time series.
     """
-    partial_get_feats = partial(_get_feats, freq=freq, scale=scale,
-                                features=features, dict_freqs=dict_freqs)
-
-    with Pool(threads) as pool:
-        ts_features = pool.starmap(partial_get_feats, ts.groupby('unique_id'))
-
-    ts_features = pd.concat(ts_features).rename_axis('unique_id')
-    ts_features = ts_features.reset_index()
-
-    return ts_features
-
-################################################################################
-#### MAIN WIDE FUNCTION ########################################################
-################################################################################
-
-def _get_feats_wide(index,
-                    ts,
-                    scale = True,
-                    features = [acf_features, arch_stat, crossing_points,
-                                entropy, flat_spots, heterogeneity, holt_parameters,
-                                lumpiness, nonlinearity, pacf_features, stl_features,
-                                stability, hw_parameters, unitroot_kpss, unitroot_pp,
-                                series_length, hurst]):
-    seasonality = ts['seasonality'].item()
-    y = ts['y'].item()
-    y = np.array(y)
-
-    if scale:
-        y = scalets(y)
-
-    c_map = ChainMap(*[dict_feat for dict_feat in [func(y, seasonality) for func in features]])
-
-    return pd.DataFrame(dict(c_map), index = [index])
-
-def tsfeatures_wide(ts: pd.DataFrame,
```

Comment:
These functions seem to be missing as well

Reply:
include both the wide feature funcs